packages/block-library/src/image/edit.js

```diff
@@ -330,40 +354,48 @@ export function ImageEdit( {
 	);
 	const placeholder = ( content ) => {
 		return (
-			<Placeholder
-				className={ clsx( 'block-editor-media-placeholder', {
-					[ borderProps.className ]:
-						!! borderProps.className && ! isSingleSelected,
-				} ) }
-				withIllustration
-				icon={ lockUrlControls ? pluginsIcon : icon }
-				label={ __( 'Image' ) }
-				instructions={
-					! lockUrlControls &&
-					__(
-						'Upload an image file, pick one from your media library, or add one with a URL.'
-					)
-				}
-				style={ {
-					aspectRatio:
-						! ( width && height ) && aspectRatio
-							? aspectRatio
-							: undefined,
-					width: height && aspectRatio ? '100%' : width,
-					height: width && aspectRatio ? '100%' : height,
-					objectFit: scale,
-					...borderProps.style,
-					...shadowProps.style,
-				} }
-			>
-				{ lockUrlControls ? (
-					<span className="block-bindings-media-placeholder-message">
-						{ lockUrlControlsMessage }
-					</span>
-				) : (
-					content
-				) }
-			</Placeholder>
+			<>
+				{ placeholderResizeListener }
+				<Placeholder
+					className={ clsx( 'block-editor-media-placeholder', {
+						[ borderProps.className ]:
+							!! borderProps.className && ! isSingleSelected,
+					} ) }
+					icon={
+						isLargeContainer &&
+						( lockUrlControls ? pluginsIcon : icon )
+					}
+					withIllustration={
+						! isSingleSelected || ! isLargeContainer
+					}
+					label={ isLargeContainer && __( 'Image' ) }
+					instructions={
+						! lockUrlControls &&
+						isLargeContainer &&
+						__(
+							'Upload an image file, pick one from your media library, or add one with a URL.'
+						)
+					}
+					style={ {
+						aspectRatio:
+							! ( width && height ) && aspectRatio
+								? aspectRatio
+								: undefined,
+						width: height && aspectRatio ? '100%' : width,
+						height: width && aspectRatio ? '100%' : height,
+						objectFit: scale,
+						...borderProps.style,
+						...shadowProps.style,
+					} }
+				>
+					{ lockUrlControls && isLargeContainer && (
+						<span className="block-bindings-media-placeholder-message">
+							{ lockUrlControlsMessage }
+						</span>
```

Comment:
```suggestion
					{ lockUrlControls &&
						isLargeContainer &&
						lockUrlControlsMessage }
```

Since this class name is no longer in use, the `span` tag itself is no longer necessary.